src/index.html

```diff
@@ -8,6 +8,7 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://smstestbed.nist.gov" rel="preconnect"> <!-- blind optimization -->
```

Comment:
I tested it. When the ng server isn't on the same computer as the browser, then TLS is already finished by the time it first requests a resource. That resource is the bottle neck, so it decreases load time by 100-300 ms.